gen-ai/orchestrator-server/src/main/python/server/src/gen_ai_orchestrator/services/langchain/rag_chain.py

```diff
@@ -166,12 +173,26 @@ def create_rag_chain(query: RagQuery) -> ConversationalRetrievalChain:
         index_name=query.document_index_name,
     )
 
+    # TODO to be removed after tests of flashrank rerank
```

Comment:
Explain that we are creating a compressor based on env variable for experimental purposes